src/pages/Components/Radio/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Radio'
+order: 8
+---
+
+> Checkboxes are used to **select only one option in a list**. They usually find their place in forms.
+
+## Preview
+
+<preview path="src/pages/Components/Radio/previews/intro" nude="true"></preview>
+
+
+## Standard radio states :
+
+### Hover
+
+When the radio is hovered due to user navigation.
+
+<preview path="src/pages/Components/Radio/previews/radio-hover-state" nude="true"></preview>
+
+### Focus
+
+When the radio is in focus. It is generally triggered when the user clicks or taps on an element or selects it with the keyboard's "tab" key.
+
+<preview path="src/pages/Components/Radio/previews/radio-focus-state" nude="true"></preview>
+
+### Disabled
+When the radio field is inactive or inaccessible. It is usually used when the input, for some reason, is forbidden or blocked to be edited.
+
+<preview path="src/pages/Components/Radio/previews/radio-disabled-state" nude="true"></preview>
```

Comment:
There is not an outline on the disable state.
Also, can we have a disable cursor on the hover?